### PR TITLE
WIP: extract UI in its own addon/chart

### PR DESCRIPTION
**Pull request by juliangieseke**
related:
https://github.com/mesosphere/kommander/pull/1770
https://github.com/mesosphere/kubernetes-base-addons/pull/143

UI chart will only live in https://github.com/mesosphere/kommander/tree/master/chart/kommander-ui

This removed UI from opsportal chart.

there are some open questions
- [ ] given the ui is now detached from opsportal, how to handle cases where no UI is present?
- [ ] we need to clarify a couple support matrix questions (WIP)
- [ ] UI addon needs to be added to kubernetes-base-addons repo and enabled by default before we can merge this

### CODE REVIEW

File: stable/opsportal/values.yaml [hunk 1]
```diff
@@ -16,40 +16,10 @@ secrets:
...
-      traefik.ingress.kubernetes.io/priority: "1"
-      traefik.ingress.kubernetes.io/auth-type: forward
-      traefik.ingress.kubernetes.io/auth-url: http://traefik-forward-auth-kubeaddons.kubeaddons.svc.cluster.local:4181/
-      traefik.ingress.kubernetes.io/auth-response-headers: X-Forwarded-User
-
 opsportalRBAC:
   enabled: true
-  path: /ops/portal
-  graphqlPath: /ops/portal/graphql
+  path: /ops/portal/dashboard
```
Comment (juliangieseke) [inline stable/opsportal/values.yaml]: TBD!

**Comment by shaneutt** [comment]
@juliangieseke do you need any assistance from @mesosphere/sig-ksphere-catalog to move this one forward?

**Comment by juliangieseke** [comment]
Not on this one. Maybe in base-addons. In general, theres a lot of things to clarify that are not directly related to code. So thanks for offer, but I think no. 

Does the PR bother you? I can close it until we actually want to merge (repoen then).

**Comment by shaneutt** [comment]
> Not on this one. Maybe in base-addons. In general, theres a lot of things to clarify that are not directly related to code. So thanks for offer, but I think no.

Ok! No problem :vulcan_salute: 

> Does the PR bother you? I can close it until we actually want to merge (repoen then).

No, I just make it a goal to check in with open PR's regularly to help keep people unblocked, no problem leaving it open while you're preparing to finish it up. :+1:

**Comment by shaneutt** [comment]
@juliangieseke just pinging, anything needed here?

**Comment by juliangieseke** [comment]
Effort is paused, closing.

**Pull request closed without merging**